### DISCUSSION TITLE
Practical series 4 - Break out room 2

### DISCUSSION THREAD

**Discussion by Rawizard** [IBAHCM/RPiR#262] [Archive]
Breakout room 2 members 

Jamie Crowther - @Rawizard
Will McLellan - @Will-214
Alexa Roditi - @AlexaRoditi
Yifei Peng - @mikoV97

**Comment by Rawizard** [comment]
My Practical series 4 repo: 

https://github.com/IBAHCM/RawizardBCT.git

**Comment by AlexaRoditi** [comment]
Practical 4 Repo (Alexa Roditi):
https://github.com/IBAHCM/AlexaRoditiBCI

**Comment by mikoV97** [comment]
My practical 4 repo (Yifei Zeng): https://github.com/IBAHCM/YifeiZengBCI

**Comment by Rawizard** [comment]
Hey @mikoV97 & @AlexaRoditi ,

You both need to give us access to your repo, either by adding the class or us individually in the manage access tab!

**Reply by mikoV97** [reply]
> Hey @mikoV97 & @AlexaRoditi ,
> 
> You both need to give us access to your repo, either by adding the class or us individually in the manage access tab!

Thanks for reminding, I forgot and just had it fixed!

**Comment by Will-214** [comment]
https://github.com/IBAHCM/Will214BCI.git
